library/coretests/tests/panic/location.rs

```diff
@@ -29,3 +29,11 @@ fn location_const_column() {
     const COLUMN: u32 = CALLER.column();
     assert_eq!(COLUMN, 40);
 }
+
+#[test]
+fn location_debug() {
+    let f = format!("{:?}", Location::caller());
+    assert!(f.contains("location.rs"));
```

Comment:
```suggestion
    assert!(f.contains("location.rs"));
    assert!(!f.contains("\0"));
    assert!(!f.contains("\\0"));
```

Reply:
I immediately clicked 'commit suggestion', but actually: i think this might cause funny failures:

If someone works on windows in a directory that starts with a `0`, the file name might be `"C:\\Users\\0abc\\Rust\\..."`, which will contain `\\0`.